crossbuilder

```diff
@@ -671,8 +751,22 @@ check_for_container_network() {
         sleep 1
     done
     if [ $NETWORK_UP -ne 1 ] ; then
-        echo -e "${ERROR_COLOR}Container is not connected to the Internet.${NC}"
-        exit 1
+        # On manjaro the 16.04 containers can't get ip4 address.
+        # This can be corrected by running 'dhclient' in container.
+        exec_container_root "dhclient"
```

Comment:
But this means that in manjaro you have to wait at least 10 seconds, right? Or are there cases when dhclient is not needed?

Does it make sense to reduce both loops to 5 iterations each, so that Manjaro users won't have to wait that long? (and it would still remain up to 10 seconds for Ubuntu users)